README.md

```diff
@@ -47,13 +47,28 @@ We recommend using the latest release of [NGC's PyTorch container](https://catal
     ```
     The above instructions create two conda environments: `flagscale-train` and `flagscale-inference`, which contain the dependency environments for training and inference, respectively.
 
-3. Install the packages with customized extensions:
+3. Unpatch the backend code adaptation of FlagScale
+    ```
+    cd FlagScale
+    python tools/patch/unpatch.py --backend Megatron-LM
+    python tools/patch/unpatch.py --backend vllm
+    ```
+
+4. Install the packages with customized extensions:
     ```sh
-    cd vllm
+    cd FlagScale/third_party/vllm
     pip install .
-
+    
+    cd FlagScale
     pip install -e ./megatron-energon
-    cp -r megatron-energon/src/megatron/energon megatron/megatron
```

Comment:
Can we move energon into third_party like Megatron-LM and vllm?

Reply:
Sure, this will be implemented in the next pull request (PR).